src/components/ReorderableListCore/useReorderableListCore.ts

```diff
@@ -175,6 +177,23 @@ export const useReorderableListCore = <T>({
         })
         .onUpdate(e => {
           if (state.value !== ReorderableListState.RELEASED) {
+            // `dragDirection` determines if the user intends to autoscroll.
+            // We only update it if a threshold is passed to account for noise,
+            // especially while holding a dragged item within the scroll zone.
+            const direction = e.velocityY > 0 ? 1 : -1;
+            if (direction !== dragDirection.value) {
+              if (lastDragDirectionPivot.value === null) {
+                lastDragDirectionPivot.value = e.absoluteY;
```

Comment:
This case does *not* set the `dragDirection` because the delta has not been determined yet.